docs-src/dev-guide/sdk-features.md

```diff
@@ -0,0 +1,571 @@
+
+### Core development
+
+All SDKs support the fundamental ability to define functions as either Workflows or Activities.
+
+<!-- [Yes](/dev-guide/go/foundations#develop-workflows) -->
+
+| Feature                                                 | Go  | Java | TypeScript | Python | .Net | PHP |
+| ------------------------------------------------------- | --- | ---- | ---------- | ------ | ---- | --- |
+| [Workflow Definitions](/workflows#workflow-definition)  | Yes | Yes  | Yes        | Yes    | Yes  | Yes |
+| [Activity Definitions](/activities#activity-definition) | Yes | Yes  | Yes        | Yes    | Yes  | Yes |
+
+A Workflow Definition is essentially Workflow code, which orchestrates the execution of Activities, persisting the results.
+[Learn more](/workflows#workflow-definition) in the concepts.
+
+An Activity Definition is a normal function or method that defines a single, well-defined action (either short or long running), such as calling another service, transcoding a media file, or sending an email message.
+For more information, see [Activity Definition](/activities#activity-definition) in the concept page.
+
+## Auth (SSL/mTLS)
+
+| Feature                         | Go  | Java | TypeScript | Python | .Net | PHP |
+| ------------------------------- | --- | ---- | ---------- | ------ | ---- | --- |
+| [Auth (SSL/mTLS)](#auth-ssltls) | Yes | Yes  | Yes        | Yes    | Yes  | Yes |
+
+Temporal SDKs support authenticating and authorizing client API calls to the Temporal server using SSL and mTLS.
+SSL (Secure Sockets Layer) and its successor TLS (Transport Layer Security) allow encrypted communication between the SDK client and the Temporal server over a network.
+mTLS (mutual TLS) is a variant where both client and server authenticate each other by verifying the provided certificates.
+This ensures secure identification and access control for API calls made from the SDK clients.
+Configuring SSL or mTLS authentication provides network encryption and protects against man-in-the-middle attacks when communicating with the Temporal server.
+
+## Built-in handler list Query
+
+| Feature                                                     | Go | Java | TypeScript | Python | .Net | PHP |
+| ----------------------------------------------------------- | -- | ---- | ---------- | ------ | ---- | --- |
+| [Built-in handler list query](#built-in-handler-list-query) | No | No   | No         | No     | No   | No  |
+
+The built-in handler list Query in Temporal allows retrieving the list of registered Signal and Query handlers for a Workflow.
+
+When this Query is sent to a Workflow, the response contains a JSON serialized list of the following information about each handler:
+
+- Handler type: either Signal or Query
+- Handler name
+- Workflow function that handles the Signal or Query
+
+This can be useful for debugging or introspecting which Signal or Query a Workflow currently supports.
+
+## Built-in Temporal CLI dev server runner
```

Comment:
This feature is not worth top-billing. It's just a tiny utility that downloads/runs the CLI which anyone can do.